addOns/paramminer/src/main/java/org/zaproxy/addon/paramminer/ExtensionParamMiner.java

```diff
@@ -90,7 +91,7 @@ public void unload() {
         super.unload();
     }
 
-    private ParamMinerPanel getParamMinerPanel() {
+    public ParamMinerPanel getParamMinerPanel() {
```

Comment:
Can this be protected instead? We try not to make GUI elements easily accessible to other extensions as that can get _very_ messy ;)

Reply:
There are a lot of changes incoming actually :). I'll add this to my TODO.

Reply:
Made it private